src/Makefile

```diff
@@ -1,5 +1,15 @@
 INCLUDE_DIRS = ../inc
 CFLAGS += -O2
 
+blst:
+	cd ../blst; \
+	git apply < ../blst_sha.patch; \
```

Comment:
dependence on git here is interesting - is depending on patch maybe more conservative? up to you

Reply:
We're already dependent on git for the submodule, so this seems okay to me.